packages/kit/test/apps/basics/test/server.test.js

```diff
@@ -473,11 +473,6 @@ test.describe('Static files', () => {
 		expect(await r2.json()).toEqual({ works: true });
 	});
 
-	test('Filenames are case-sensitive', async ({ request }) => {
-		const response = await request.get('/static.JSON');
-		expect(response.status()).toBe(404);
-	});
```

Comment:
why was this test deleted?

Reply:
it looks like it doesn't need to be actually. I just restored it

Reply:
Right, but many servers do, so it's essential that we do too — if we can't get Vite to be strict about casing it's a showstopper for this PR